README.md

```diff
@@ -63,7 +70,8 @@ How would I go about accessing the string "pizza" in the above array?
 Describe the rules of scope in JavaScript.
 
 Your Answer:
-```text
+```
+Scope defines where the variable is accessible. If variable is declared outside of function then it is accessible globally through out the code whereas a local variable which is inside of a function is only available to access in that function. If variable is not declared inside the function it will be accessible through out the code.
 ```
```

Comment:
:+1: but remember you can have nested functions, so you can have varying levels of scope that are not global.